### DISCUSSION TITLE
how do we provide the sourcemaps to sentry?

### DISCUSSION THREAD

**Discussion by adiig7** [nuxt-community/sentry-module#703] [Q&A] [answered]
i have nuxt2 application and integrated sentry-module but I want to provide the sourcemaps to get better stack tracing, how do I enable that?

**Comment by adiig7** [comment]
@rchl could you please help here?

**Comment by rchl** [comment]
There is documentation at https://sentry.nuxtjs.org/guide/releases

**Reply by adiig7** [reply, marked as answer]
found, thanks!
